open-metadata-implementation/access-services/asset-catalog/asset-catalog-server/src/main/java/org/odpi/openmetadata/accessservices/assetcatalog/handlers/RelationshipHandler.java

```diff
@@ -79,7 +81,7 @@ public void setSecurityVerifier(OpenMetadataServerSecurityVerifier securityVerif
         if (relationshipBetweenEntities != null) {
 
             securityVerifier.validateUserForRelationshipRead(userId,
```

Comment:
This is not the right method to call - this is the call that is reserved for the OMRS to validate calls to the repository.

Reply:
Should I create another method in the Security Verifier? The Asset Catalog OMAS can not use the validation for Asset because it searches also for other types.

Reply:
The supportedZones are only for Assets.  So are the asset security methods.
The security works in layers - the platform services validate that the user has access to the server and the service.  Then the handlers validate for security that is specific to the open metadata type.  Currently these are only for Asset and Connection.  Later we will add for Personal Profile.  However, most types do not need specialist validation.  Finally the repository services validate that they user has access to the instance in the repository.

Reply:
Hi! I have updated this PR. For now, I'm using only invalid parameter handler.
I removed the unnecessary validations.

Reply:
Hi, Mandy! Can you take a look on the latest changes that I've done?